src/lib/VoicemeeterConnector.ts

```diff
@@ -302,11 +294,17 @@ export default class Voicemeeter {
 	 * Returns the installed voicemeeter version
 	 */
 	private getVoicemeeterVersion = () => {
-		const versionPtr = new LongArray(1) as any;
+		const versionPtr = [0];
 		if (libVM.VBVMR_GetVoicemeeterVersion(versionPtr) !== 0) {
 			throw new Error("running failed");
 		}
-		return versionPtr;
+		// For info on this see: https://github.com/mirror/equalizerapo/blob/53d885f7f1a097b457e17a5206b7d60f647877a8/VoicemeeterClient/VoicemeeterRemote.h#L122-L135
+		const version =
+			`${(versionPtr[0] & 0xff000000) >> 24}` +
+			`.${(versionPtr[0] & 0x00ff0000) >> 16}` +
+			`.${(versionPtr[0] & 0x0000ff00) >> 8}` +
+			`.${versionPtr[0] & 0x000000ff}`;
+		return version;

```

Comment:
idk if you intentionally returned the 'pointer' here, so I changed it to return the string version ([more info here](https://github.com/mirror/equalizerapo/blob/53d885f7f1a097b457e17a5206b7d60f647877a8/VoicemeeterClient/VoicemeeterRemote.h#L122-L135))

Reply:
makes sense for me.